data-cube-curation/templates/deployment.yaml

```diff
@@ -0,0 +1,152 @@
+{{- $fullName := include "data-cube-curation.fullname" . -}}
+{{- $sparqlSecretName := include "data-cube-curation.sparql.secretName" . -}}
+{{- $gitlabSecretName := include "data-cube-curation.gitlab.secretName" . -}}
+{{- $s3SecretName := include "data-cube-curation.s3.secretName" . -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ $fullName }}
+  labels:
+    {{- include "data-cube-curation.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "data-cube-curation.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "data-cube-curation.selectorLabels" . | nindent 8 }}
+    spec:
+    {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - name: http
+              containerPort: 3000
+              protocol: TCP
+          livenessProbe:
```

Comment:
An `initialDelaySeconds` probably makes sense here, to give the pod some time to start up